client/src/plugins/camunda-plugin/shared/webAppUrls.js

```diff
@@ -0,0 +1,48 @@
+/**
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH
+ * under one or more contributor license agreements. See the NOTICE file
+ * distributed with this work for additional information regarding copyright
+ * ownership.
+ *
+ * Camunda licenses this file to you under the MIT; you may not use this file
+ * except in compliance with the MIT License.
+ */
+
+import { forEngineRestUrl } from './WellKnownAPI';
+
+export async function determineCockpitUrl(engineUrl) {
+  try {
+    const cockpitUrl = await forEngineRestUrl(engineUrl).getCockpitUrl();
+
+    if (cockpitUrl) {
+      console.debug(`Using cockpit url from well known endpoint: ${engineUrl}`);
```

Comment:
I'd rather use `debug` dependency consistently instead of `console.debug` which is the same as `console.log`.

Reply:
I exchanged those calls.